src/nina_vesseltun_ex/ex03_project/test_median.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Nina Mariann Vesseltun'
+__email__ = 'nive@nmbu.no'
+
+
+import pytest
+
+
+def median(data):
+    """
+    Returns median of data.
+
+    :param data: An iterable of containing numbers
+    :return: Median of data
+    """
+    try:
+        sorted_data = sorted(data)
+        num_elements = len(sorted_data)
+        if num_elements % 2 == 1:
+            return sorted_data[num_elements // 2]
+        else:
+            return (
+                sorted_data[num_elements // 2 - 1] + sorted_data[num_elements
+                                                                 // 2]) / 2
+    except ValueError as err:
```

Comment:
Only IndexError is raised here. The except keyword won't catch the ValueError exception. You have to check if the list is empty then raise ValueError inside